README.md

```diff
@@ -63,34 +65,44 @@ However, we're proposing unlimited-precision decimal instead, for the following
-- How (if at all) should we represent rounding modes? [#19](https://github.com/littledan/proposal-bigdecimal/issues/19)
-- How should operations which necessarily round, e.g., division, be supported? [#13](https://github.com/littledan/proposal-bigdecimal/issues/13)
-- What standard library functions should exist? [#14](https://github.com/littledan/proposal-bigdecimal/issues/14)
-
-We'd especially encourage you to help us answer these and other questions by [contributing documentation about use cases you care about](https://github.com/littledan/proposal-bigdecimal/issues/3).
+BigDecimal is generally analogous to BigInt, complete with:
+- Literal syntax: `123.456d` is a BigDecimal value ([#7](https://github.com/littledan/proposal-bigdecimal/issues/7))
+- Operator overloading: `.1d + .2d === .3d`
+
+Data model:
+- BigDecimal represents a mathematical, "normalized" ([#26](https://github.com/littledan/proposal-bigdecimal/issues/26)) base 10 decimal, of unlimited size ([#8](https://github.com/littledan/proposal-bigdecimal/issues/8)).
+    - For example, `2d` is exactly the same value as `2.00d` ([#11](https://github.com/littledan/proposal-bigdecimal/issues/11))
+    - If trailing zeroes or other kinds of magnitude/precision need to be represented separately from the BigDecimal
+    - There is no Infinity, -0, NaN, etc; error cases lead to exceptions, just like BigInt, and `-0d` is `0d` ([#9](https://github.com/littledan/proposal-bigdecimal/issues/9))
+- A new primitive type, not an object: `typeof 1d === "bigdecimal"`
+    - There can still be methods on `BigDecimal.prototype` due to the magic of wrappers, just like Number.
+
+Operator semantics:
+- Operators which can be calculated exactly are defined to return their exact answer (`+`, `-`, `*`, `%`, etc.) ([#10](https://github.com/littledan/proposal-bigdecimal/issues/10))
+- Operators which would need to round instead have methods to support them, described below ([#13](https://github.com/littledan/proposal-bigdecimal/issues/13))
+- Bitwise operators are not supported, as they don't logically make sense on the BigDecimal domain ([#20](https://github.com/littledan/proposal-bigdecimal/issues/20))
+- Use explicit casts when you need to do a calculation involving different numerical types. Otherwise, a TypeError is thrown, like for BigInt+Number.
+- Comparison with `===` compares two BigDecimals for mathematical equality, and returns false if comparison is with another type; comparison with `==`, `<`, etc can compare BigDecimal with any numerical type
+
+BigDecimal methods for calculation: ([#14](https://github.com/littledan/proposal-bigdecimal/issues/14))
+- `BigDecimal.prototype.round()` rounds a BigDecimal, based on an options bag with the following parameters:
+    - `roundingMode`: Rounding mode, with exact set of values TBD, maybe including `"up"`, `"down"`, `"half-up"`, `"half-down"`, `"half-even"` (more?). There is no default; this must be explicitly provided
```

Comment:
Why no `"half-up"` as default?

Reply:
This is very debatable, but my idea here was to pedantically force people to make a choice. (Another default would be "half-even".) If we want to choose a default, I think it'd be best if we could enumerate lots of use cases and what rounding modes make sense for them, to make the decision.